apps/omg_watcher/test/challenger/core_test.exs

```diff
@@ -42,57 +44,44 @@ defmodule OMG.Watcher.Challenger.CoreTest do
         txindex2: 0,
         oindex2: 1,
         cur12: <<0::160>>,
-        newowner1: "alice",
+        newowner1: <<1::160>>,
```

Comment:
in omg_api/test/support/test_helper.ex is stable_alice and stable_bobo

Reply:
Ah, I changed it because we added some constraint for address length... here I don't want/care about real addresses, will leave to not make this test depend on fixtures